src/lib/components/error-pages/ErrorPageStyle.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { spacing, fontSize } from '../../style/theme';
+import { getThemePropSelector } from '../../utils';
+
+export const ErrorPageContainer = styled.div`
+  background-color: ${getThemePropSelector('backgroundLevel1')};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
nit: I'm not super sure about this `height: 100vh;` I think in theory it should match the height of the parent. (e.g if we display this component below the navbar a scrollbar will be displayed, we already have this pb so this is nitpicking...)

Reply:
You are right. height: 100% is enough.